FinniversKit/Sources/Components/Ribbon/IconRibbonView.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  Copyright © FINN.no AS, Inc. All rights reserved.
+//
+
+import UIKit
+
+public class IconRibbonView: UIView {
+
+    // MARK: - Private properties
+
+    private lazy var iconImageView: UIImageView = {
```

Comment:
I guess we don't need the multi-line declaration

Reply:
Yup, forgot about this one😅 Fixing